tango_simlib/utilities/fandango_json_parser.py

```diff
@@ -18,6 +18,7 @@
 
 from PyTango._PyTango import CmdArgType, AttrDataFormat
 
+from tango_simlib.utilities.base_parser import Parser
 MODULE_LOGGER = logging.getLogger(__name__)
```

Comment:
- [ ] Add a newline here.

Reply:
- [ ] Fix this import statement `from helper_module import json_load_byteified` on line 17.